src/GraphQL.Tests/GraphQL.Tests.xproj

```diff
@@ -11,9 +11,11 @@
     <BaseIntermediateOutputPath Condition="'$(BaseIntermediateOutputPath)'=='' ">.\obj</BaseIntermediateOutputPath>
     <OutputPath Condition="'$(OutputPath)'=='' ">.\bin\</OutputPath>
   </PropertyGroup>
-
   <PropertyGroup>
     <SchemaVersion>2.0</SchemaVersion>
   </PropertyGroup>
+  <ItemGroup>
```

Comment:
What version of VS are you using?  Could just be a mis-match between the two of us.  I am running `14.0.25425.01 Update 3`.